pyomo/environ/__init__.py

```diff
@@ -58,6 +58,7 @@ def _do_import(pkg_name):
     'pyomo.contrib.multistart',
     'pyomo.contrib.petsc',
     'pyomo.contrib.preprocessing',
+    'pyomo.contrib.ampl_function_demo',
```

Comment:
Could you move this to the top of the set to keep the contrib packages in alphabetical order?